utils.py

```diff
@@ -136,6 +136,21 @@ def get_build():
     sys.exit(1)
 
 
+def get_repo_ref(config, tree_name):
+    for tree in config["trees"]:
+        if tree["name"] == tree_name:
+            return tree["git_repo"], tree["git_ref"]
+
+
+def get_llvm_versions(config, tree_name):
+    llvm_set = set()
```

Comment:
s/llvm_set/llvm_versions/ or just `versions` (we know it's a set, avoid hungarian notation)

Reply:
Sounds good, I'll do `llvm_versions`.